our_files/custom.js

```diff
@@ -426,14 +424,13 @@ var deVSeriesLink = function () {
 
 //Detail View Tasks -- ITEM_STATUS tasks
 var ILLIfCheckedOut = function () {
```

Comment:
Can you put in place a new comment with the format:

/*
Purpose: ???
Example URL: ???

Desktop Incoming Markup: TBD
Desktop Outgoing Markup: TBD

Mobile Incoming Markup: TBD
Mobile Outgoing Markup: TBD
*/

So we can all more easily understand what this expects and what it is doing.